Sources/SwiftInspectorCommands/Tests/TypealiasCommandSpec.swift

```diff
@@ -66,6 +66,20 @@ final class TypealiasCommandSpec: QuickSpec {
         it("does not output the typelias identifiers") {
           expect(result?.outputMessage).toNot(contain("SomeType"))
         }
+
+        context("when a typealias is defined multiple times") {
+          it("only returns one the typealias name once") {
+            fileURL = try? Temporary.makeFile(content: """
+                                                       typealias SomeAlias = SomeType
+                                                       typealias SomeAlias = SomethingElse
+                                                       """
+                                                       )
+            result = try? TestTask.run(withArguments: ["typealias", "--path", fileURL!.path])
+
+            let lines = result?.outputMessage?.split { $0.isNewline }.count
+            expect(lines) == 1
```

Comment:
Related: let's say you had code like this:

```swift
typealias Foo = Bar

struct MyNamespace {

  typealias Foo = Bar

}
```

I would also expect SwiftInspect to return this twice, or at least print out the information for each of them when I pass `--name Foo`.

How can we handle this valid syntax better?

Reply:
I'd say if they're the same it'd be expected to pass the information back once, if we have something different with the same name though and a `--name` parameter I agree that we should pass it back twice.